pkg/wasm/cache.go

```diff
@@ -226,6 +226,7 @@ func (c *LocalFileCache) getEntry(key cacheKey) string {
 	if ce, ok := c.modules[key]; ok {
 		// Update last touched time.
 		ce.last = time.Now()
+		c.modules[key] = ce
```

Comment:
why need this?

Reply:
the `ce` is a copy of the element in map

Reply:
ce is not a point?

Reply:
yes. you can see here:https://github.com/istio/istio/pull/37578/files#diff-b4b9abc466fa6e7cad54037be486711eda1582c289c1dcbdd13aebd776442f5cL94

Reply:
how about change it to a point, wirting back looks wired.

Reply:
ok